src/style/stylize.rs

```diff
@@ -137,7 +137,7 @@ macro_rules! modifier {
 /// ```
 pub trait Stylize<'a, T>: Sized {
     #[must_use = "`bg` returns the modified style without modifying the original"]
-    fn bg(self, color: Color) -> T;
+    fn bg<S: Into<Color>>(self, color: S) -> T;
```

Comment:
S was probably a short for Style some time ago…
```suggestion
    fn bg<C: Into<Color>>(self, color: C) -> T;
```

Reply:
Resolved in https://github.com/ratatui-org/ratatui/pull/1099/commits/090fb90075cd50d6a2f1935f6506cdbd60077155